iml-services/iml-snapshot/src/main.rs

```diff
@@ -27,23 +126,66 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let pool = get_db_pool(get_pool_limit().unwrap_or(DEFAULT_POOL_LIMIT)).await?;
     sqlx::migrate!("../../migrations").run(&pool).await?;
 
+    let snapshot_client_counts: Arc<Mutex<HashMap<SnapshotId, Option<State>>>> =
+        Arc::new(Mutex::new(HashMap::new()));
+    let snapshot_client_counts_2 = snapshot_client_counts.clone();
+
+    tokio::spawn(async move {
+        // Avoid conflict with futures_util::StreamExt
```

Comment:
I think this is why traits are imported like ` use foo::Foo as _`

Reply:
Doesn't help in this instance

Reply:
We don't need the tokio::StreamExt, just the regular futures one